cumulus_library/protected_table_builder.py

```diff
@@ -1,10 +1,10 @@
 """Builder for creating tables for tracking state/logging changes"""
 
-from cumulus_library import base_table_builder, enums, study_parser
+from cumulus_library import base_table_builder, databases, enums, study_parser
 from cumulus_library.template_sql import base_templates
 
-TRANSACTIONS_COLS = ["study_name", "library_version", "status", "event_time"]
-TRANSACTION_COLS_TYPES = ["varchar", "varchar", "varchar", "timestamp"]
+TRANSACTIONS_COLS = ["study_name", "library_version", "status", "event_time", "message"]
+TRANSACTION_COLS_TYPES = ["varchar", "varchar", "varchar", "timestamp", "varchar"]
```

Comment:
i talked about adding a gentle upgrade path for this, but if we're making this part of a 3.0 breaking changeset, i'm inclined to say something like 'just remove previous lib_transaction tables'. Can be talked out of this position.